linux/LinuxProcessList.c

```diff
@@ -1045,6 +1105,14 @@ static bool LinuxProcessList_recurseProcTree(LinuxProcessList* this, const char*
             }
          }
       }
+      /* (Re)Generate the Command string, but only if the process is:
+       * - not a kernel thread, and
+       * - not a zombie or it became zombie under htop's watch, and
+       * - not a user thread or if showThreadNames is not set */
+      if (!Process_isKernelThread(proc) && (proc->state != 'Z' || lp->mergedCommand.str) &&
```

Comment:
please use a new line for every AND condition

Reply:
Agreed